using_images/s2i_images/php.adoc

```diff
@@ -124,6 +124,13 @@ The following environment variable sets its equivalent property value in the opc
 |16M
 |===
 
+ifdef::openshift-origin[]
+|`*OPCACHE_REVALIDATE_FREQ*`
+|How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.
```

Comment:
The problem I have with this wording is that it mentions `OPcache` without explaining what it is, or linking to it. Mabye you could make the word "OPcache" a link to here? https://secure.php.net/manual/en/intro.opcache.php